components/gitpod-protocol/go/gitpod-service.go

```diff
@@ -1490,9 +1490,11 @@ type UserInfo struct {
 
 // GetWorkspacesOptions is the GetWorkspacesOptions message type
 type GetWorkspacesOptions struct {
-	Limit        float64 `json:"limit,omitempty"`
-	PinnedOnly   bool    `json:"pinnedOnly,omitempty"`
-	SearchString string  `json:"searchString,omitempty"`
+	Limit                 float64 `json:"limit,omitempty"`
+	SearchString          string  `json:"searchString,omitempty"`
+	PinnedOnly            bool    `json:"pinnedOnly,omitempty"`
+	IncludeWithoutProject bool    `json:"includeWithoutProject,omitempty"`
```

Comment:
includeWithoutProject is it neccessary?

Reply:
@akosyakov it isn't, but it should be the same as in our [typescript source](https://github.com/gitpod-io/gitpod/blob/0d22d834b1c063f59db3d6af8f90460db9e128ff/components/gitpod-protocol/src/gitpod-service.ts#L382). This is just trying to reflect that, not adding anything to public API.

Reply:
again, typescript is not a source, it is completely another API